r-python-eds-lessons/environment.yml

```diff
@@ -44,4 +44,5 @@ dependencies:
       - elevation
       - hydrofunctions
       - mapboxgl
+      - nbclean
```

Comment:
hey @jvtcl i think we can pin tweepy to 3.7 now and that should work with python 3.7. if it doesn't work let me know but they just fixed it this morning.

Reply:
ok i just fixed and merged!!